system-parachains/collectives/collectives-polkadot/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -0,0 +1,190 @@
+    pub(crate) fn withdraw_asset() -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `101`
+        //  Estimated: `3593`
+        // Minimum execution time: 40_160_000 picoseconds.
+        Weight::from_parts(40_961_000, 0)
+            .saturating_add(Weight::from_parts(0, 3593))
+            .saturating_add(T::DbWeight::get().reads(1))
+            .saturating_add(T::DbWeight::get().writes(1))
+    }
+    /// Storage: `System::Account` (r:2 w:2)
+    /// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+    pub(crate) fn transfer_asset() -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `153`
+        //  Estimated: `6196`
+        // Minimum execution time: 54_831_000 picoseconds.
+        Weight::from_parts(55_110_000, 0)
+            .saturating_add(Weight::from_parts(0, 6196))
+            .saturating_add(T::DbWeight::get().reads(2))
+            .saturating_add(T::DbWeight::get().writes(2))
+    }
+    /// Storage: `System::Account` (r:3 w:2)
+    /// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+    /// Storage: `ParachainInfo::ParachainId` (r:1 w:0)
+    /// Proof: `ParachainInfo::ParachainId` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
+    /// Storage: `ParachainSystem::UpwardDeliveryFeeFactor` (r:1 w:0)
+    /// Proof: `ParachainSystem::UpwardDeliveryFeeFactor` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+    /// Storage: `PolkadotXcm::SupportedVersion` (r:1 w:0)
+    /// Proof: `PolkadotXcm::SupportedVersion` (`max_values`: None, `max_size`: None, mode: `Measured`)
+    /// Storage: `ParachainSystem::HostConfiguration` (r:1 w:0)
+    /// Proof: `ParachainSystem::HostConfiguration` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+    /// Storage: `ParachainSystem::PendingUpwardMessages` (r:1 w:1)
+    /// Proof: `ParachainSystem::PendingUpwardMessages` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+    pub(crate) fn transfer_reserve_asset() -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `264`
+        //  Estimated: `8799`
+        // Minimum execution time: 109_430_000 picoseconds.
+        Weight::from_parts(111_451_000, 0)
+            .saturating_add(Weight::from_parts(0, 8799))
+            .saturating_add(T::DbWeight::get().reads(8))
+            .saturating_add(T::DbWeight::get().writes(3))
+    }
+    /// Storage: `Benchmark::Override` (r:0 w:0)
+    /// Proof: `Benchmark::Override` (`max_values`: None, `max_size`: None, mode: `Measured`)
+    pub(crate) fn reserve_asset_deposited() -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `0`
+        //  Estimated: `0`
+        // Minimum execution time: 18_446_744_073_709_551_000 picoseconds.
+        Weight::from_parts(18_446_744_073_709_551_000, 0)
```

Comment:
quite big, but seems not only for this runtime

Reply:
this is basically `Weight::MAX` and used for unsupported instructions - in this particular case `ReserveAssetDeposited` is disabled on collectives because it doesn't support any reserve assets, it only supports DOT through teleports

Reply:
> this is basically `Weight::MAX` and used for unsupported instructions - in this particular case `ReserveAssetDeposited` is disabled on collectives because it doesn't support any reserve assets, it only supports DOT through teleports

:point_up_2:

Reply:
Maybe a DQ: why not just use `Weight::Max`?

Reply:
This is how benchmarks generate it, not sure about the reasoning why it was implemented like this.